scripts/sdk-create-pr.sh

```diff
@@ -6,11 +6,10 @@ ROOT_DIR=$(git rev-parse --show-toplevel)
 COMMIT_NAME="SDK Generator Bot"
 COMMIT_EMAIL="noreply@stackit.de"
 SDK_REPO_LOCAL_PATH="${ROOT_DIR}/sdk-repo-updated" # Comes from generate-sdk.sh
-REPO_URL_SSH="git@github.com:stackitcloud/stackit-sdk-go.git"
 REPO_BRANCH="main"
 
-if [ $# -ne 4 ]; then
-    echo "Not enough arguments supplied. Required: 'branch-name' 'commit-message' 'pr-title' 'pr-body'"
+if [ $# -ne 5 ]; then
+    echo "Not enough arguments supplied. Required: 'branch-name' 'commit-message' 'pr-title' 'pr-body' 'repo-url'"
```

Comment:
This one doesn't need to be required, right?

Reply:
The fallback is already below

Reply:
Changed condition to lt 4 as it should be at least 4 but can also be 5 :)